backend/tracim_backend/lib/core/content.py

```diff
@@ -2099,7 +2099,7 @@ def generate_folder_label(self, workspace: Workspace, parent: Content = None) ->
 
         return _("New folder {0}").format(query.count() + 1)
 
-    def _allow_empty_label(self, content_type_slug):
+    def _allow_empty_label(self, content_type_slug: str):
```

Comment:
what is the return of the function? add full typing

Reply:
done.